src/Files.Backend/Enums/ArchiveSplittingSizes.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.Backend.Enums
+{
+	public enum ArchiveSplittingSizes
+	{
```

Comment:
Can you add a comment here as you did for `ArchiveFormats` and `ArchiveCompressionLevel`?

Reply:
Fixed